src/Utils/JWT.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+
+const typeToClaim = type => {
+  const typeMap = {
+    video: 'v',
+    thumbnail: 't',
+    gif: 'g',
+  };
+
+  return typeMap[type];
+};
+
+const getSigningKey = options => {
+  if (options.keyId) {
+    return options.keyId;
+  } else if (process.env.MUX_SIGNING_KEY) {
+    return process.env.MUX_SIGNING_KEY;
+  }
+
+  throw new TypeError('Signing Key ID required');
+};
```

Comment:
Alternatively, this may be more concise.
```
 const getSigningKey = options => {
   const keyId = options.keyId || process.env.MUX_SIGNING_KEY;
   if (!keyId) {
     throw new TypeError('Signing Key ID required');
   }
   return keyId;
};
```

Reply:
Good catch!